TypeScript/serverOnly.ts

```diff
@@ -0,0 +1,45 @@
+// @ts-nocheck
+/**
+ * Checks if a command is available in a specific server.
+ *
+ * @author @D3ord3NidAm [<@1017182455926624316>]
+ * @version 1.0.0
+ * @example
+ * ```ts
+ * import { commandModule, CommandType } from "@sern/handler";
+ * import { serverOnly } from "../plugins/serverOnly";
+ * export default commandModule({
+ *   type: CommandType.Both,
+ *   plugins: [serverOnly(["guildId"], failMessage)], // fail message is the message you will see when the command is ran in the wrong server.
+ *   description: "command description",
+ *   execute: async (ctx, args) => {
+ *     // your code here
+ *   },
+ * });
+ * ```
+ */
+
+import { CommandType, EventPlugin, PluginType } from "@sern/handler";
+
+export function serverOnly(
+  guildId: string[],
+  failMessage?: string
```

Comment:
You may add a default message here, may prove to be cleaner than checking in line 34.
```ts
failMessage = "I am unable to comply with your command."
```

Reply:
In my code, that worked as a default.